tests/integration_tests/test_tensorflow.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+
+import pytest
+
+try:
+    import tensorflow as tf
+    _available = True
+except ImportError:
+    _available = False
+
+
+import optuna
+from optuna.integration import TensorFlowPruningHook
+from optuna.testing.integration import DeterministicPruner
+
+
+def fixed_value_input_fn():
```

Comment:
Please add a type hint to `fixed_value_input_fn`.

Reply:
I added a type hint as below:

```python
# type: () -> typing.Tuple[typing.Dict[str, tf.Tensor], tf.Tensor]
```